clusterloader2/pkg/measurement/common/slos/slo_measurement.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package slos
+
+import (
+	"fmt"
+	"k8s.io/klog"
+	"k8s.io/perf-tests/clusterloader2/pkg/errors"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	sloMeasurementName = "SloMeasurement"
+)
+
+func init() {
```

Comment:
Please refactor to how it is done in:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/bundle/test_metrics.go#L33

Reply:
One question on this implementation. I did this with index based 

https://github.com/kubernetes/perf-tests/pull/1479/files#diff-c3efdfd3ea0e1e25c16d69a1a14c66aaR38-R47

but wanted to know if it's the right way?

Reply:
It's roughly it - more targeted comments below.